### ISSUE TITLE
[bug] BertWordPieceTokenizer, different tokenization than transformers around char `–`

### ISSUE THREAD

**Issue by knuser** [huggingface/tokenizers#115]
`BertWordPieceTokenizer` from *tokenizers* produces different encoding than `BertTokenizer` from *transformers* around char `–`. Tokenization around out-of-ASCII chars could be affected same way.

```
content = "first epoch after the  Cretaceous–Paleogene extinction event 66 million years ago"
tokenizer = BertTokenizer.from_pretrained('bert-base-cased')
fast_tokenizer = BertWordPieceTokenizer('./bert-base-cased/vocab.txt', lowercase=False, clean_text=False, strip_accents=False, add_special_tokens=False)

print(fast_tokenizer)
print(tokenizer.tokenize(content))
print(fast_tokenizer.encode(content).tokens)
```
Output:
```
Tokenizer(vocabulary_size=28996, model=BertWordPiece, add_special_tokens=False, unk_token=[UNK], sep_token=[SEP], cls_token=[CLS], clean_text=False, handle_chinese_chars=True, strip_accents=False, lowercase=False, wordpieces_prefix=##)
['first', 'e', '##po', '##ch', 'after', 'the', 'Cretaceous', '–', 'Pa', '##leo', '##gene', 'extinction', 'event', '66', 'million', 'years', 'ago']
['first', 'e', '##po', '##ch', 'after', 'the', 'Cretaceous', '##–', '##P', '##ale', '##ogen', '##e', 'extinction', 'event', '66', 'million', 'years', 'ago']
```

**Comment by knuser** [comment]
I should mention here, that I did not check what output will produce original bert tokenizer from https://github.com/google-research/bert
So I'm not sure which version corresponds to original bert code, but I'm counting on exact same output.

**Comment by n1t0** [comment]
This has been fixed by #134, thanks to @Mansterteddy. Feel free to reopen if you notice anything!
